spec/features/interview_spec.rb

```diff
@@ -1,6 +1,11 @@
 require "rails_helper"
+require 'selenium-webdriver'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.